Named_VTK.f90

```diff
@@ -0,0 +1,122 @@
+subroutine Fastscape_Named_VTK (f, vex, istep, foldername, k)
+
```

Comment:
What is this file used for? I don't think it should be on the root directory

Reply:
This is a fastscape file to write visualizations within the specified ASPECT output folder, and show a few more fields than the default fastscape function. I've made a pull request for them to include this within fastscape so hopefully we can remove it from here altogether.